docs/pages/setup/reference/backends.mdx

```diff
@@ -574,10 +574,25 @@ sessions in a GCS bucket.
 teleport:
   storage:
       # Path to GCS to store the recorded sessions in.
-      audit_sessions_uri: "gs://Example_TELEPORT_STORAGE/records"
-      credentials_path: /var/lib/teleport/gcs_creds
+      audit_sessions_uri: 'gs://$BUCKET_NAME/records?projectID=$PROJECT_ID&credentialsPath=$CREDENTIALS_PATH'
 ```
 
+We recommend creating a bucket in `Dual-Region` mode with the `Standard` storage class to ensure cluster performance and high availability.
+Replace the following variables in the above example with your own values:
+- `$BUCKET_NAME` with the name of the desired GCS bucket. If the bucket does not exist it will be created.
+  Please ensure the following permissions are granted for the given bucket:
+  - `storage.buckets.get`
+  - `storage.objects.create`
+  - `storage.objects.get`
+  - `storage.objects.list`
+  - `storage.objects.update`
+
+  If the bucket does not exist, please also ensure that the `storage.buckets.create` permission is granted.
+
+- `$PROJECT_ID` with a GCS-enabled GCP project.
+
+- `$CREDENTIALS_PATH` with the path to a JSON-formatted GCP credentials file configured for a service account applicable to the project.
```

Comment:
Do we already discuss how to create the service account and what permissions it should have? If not, we should add it.

Reply:
I didn't go into this here because we don't do it for other backends like DynamoDB and S3. We go into this further here: https://goteleport.com/docs/setup/deployments/gcp/

Reply:
We've had some unhappy users lately who didn't know what permissions were needed to use Teleport, so I'd like to see it documented somewhere. If you don't think it fits here feel free to file a separate issue.

Reply:
I did include the permissions required for this very reason. I thought you were referring specifically on how to create a service account & download the key with screenshots, which the above guide explains.